addon/components/frost-button.js

```diff
@@ -69,11 +74,22 @@ export default Component.extend(PropTypeMixin, {
     text: PropTypes.string,
     title: PropTypes.string,
     type: PropTypes.string,
-    vertical: PropTypes.bool
+    vertical: PropTypes.bool,
+
+    // state
```

Comment:
Ok, so the proposal here will be to keep the sections (under the assumption they remain at a low number - which seems reasonable).  Once again, I'll put this out to #frost-design for any comments, then close the associated issue.